v4/source/users.yaml

```diff
@@ -1268,3 +1268,74 @@
         '404':
           $ref: '#/responses/NotFound'
 
+  /users/tokens/disable:
+    post:
+      tags:
+        - users
+      summary: Temporarily disable a user access token
```

Comment:
Maybe shorten the summaries to `Disable personal access token` and `Enable personal access token`, they go into a sidebar so shorter the better

Also the documentation should refer to them as "personal access tokens", while in code they are "user access tokens"